intranet/static/js/eighth/statistics.js

```diff
@@ -0,0 +1,65 @@
+$(document).ready(function() {
+    if (raw_data.keys.length > 0) {
+        $("#members-chart").show();
+        var canvas = $("#members-chart")[0];
+        var data = {
+            labels: raw_data.keys,
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'raw_data' is not in camel case.](https://www.codacy.com/app/tjhsst-ion/ion/file/2620588210/issues/source?bid=3212528&fileBranchId=3299315#l6)